TypeScript.YAML-tmLanguage

```diff
@@ -466,12 +466,12 @@ repository:
 
   case-clause:
     name: case-clause.expr.ts
-    begin: '(?<!\.)\b(case|default)\b'
+    begin: '(?<!\.)\b(case|default(?=:))\b'
```

Comment:
What is the precedence here? Is this parsed as "`case` OR `default(?=:)`"? Do you only highlight `case` if it immediately followed by a colon? Can you test this out explicitly?